index.bs

```diff
@@ -779,6 +782,21 @@ spec:dom-ls; type:interface; text:Document
     dangerous and should not be used if you have Real Deal auth and csrf cookies
     used.
 
+   * '<dfn>`unsafe-include-credentials`</dfn>' All cross-origin requests to the
```

Comment:
Hm, I think that's covered by the requests link, which goes into the fetch spec, and requests covers all types of requests. I'll make a note here, however, to explain that.